### PR TITLE
added VideoQuestion Exercise

### PR DESCRIPTION
**Pull request by Toseban**
Issue #54

### CODE REVIEW

**Comment by inyono** [comment]
Thanks for the PR 🎉 I will review it tonight. Do you want to come to our dev weekly tomorrow (3pm - 5pm)?

**Comment by Toseban** [comment]
Yes I'd like to come to the dev weekly.
See you there later

File: src/components/exercises/VideoQuestion.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
+import React, {Component} from 'react';
+import {View, Text, TouchableWithoutFeedback, TouchableOpacity, Image} from "react-native";
```
Comment (inyono) [inline src/components/exercises/VideoQuestion.js]: `yarn format` pls :)

File: src/components/exercises/VideoQuestion.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+      return {highlighted};
+    });
+  };
+
+  render() {
+    return (
+      <View style={{alignItems: "center", justifyContent: "space-around", flex: 1, backgroundColor: PRIMARY}}>
+        <View style={styles.vidContainer}>
+          <TouchableWithoutFeedback onPress={this.playPause}>
+            <Video
```
Comment (inyono) [inline src/components/exercises/VideoQuestion.js]: We should probably outsource the Video part into its own component (handling replay / icon etc). This doesn't have to be part of this PR, though, I will create an issue for that.

File: src/components/exercises/VideoQuestion.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+          </TouchableWithoutFeedback>
+          <TouchableOpacity style={[styles.button, !this.state.isPaused && !this.state.isFinished ? styles.hidden : null]}
+                            onPress={this.state.isFinished ? this.replay : this.playPause}>
+            <Image source={ this.state.isFinished ? replayIcon : playIcon } style={styles.icon}/>
+          </TouchableOpacity>
+        </View>
+        <View style={{flex: 4, alignItems: "center", justifyContent: "center"}}>
+          <Text style={[DEFAULT, {marginBottom: 25}]}>{this.props.question}</Text>
+          <View>{mapIndexed((item, key) => (
+            <RoundTextButton text={item}
```
Comment (inyono) [inline src/components/exercises/VideoQuestion.js]: #106

File: src/components/exercises/VideoQuestion.js (hunk 2 above)
Comment (inyono) [inline src/components/exercises/VideoQuestion.js]: https://github.com/serlo-org/serlo-abc/issues/107

File: src/components/exercises/VideoQuestion.test.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+jest.mock('expo', () => ({
+  Video: 'Video'
+}));
+
+it('renders without crashing', () => {
+  const tree = renderer.create(
+    <VideoQuestion
+      video={require('../../assets/videos/placeholder.mp4')}
+      question="Wer ist eine Ananas? "
```
Comment (inyono) [inline src/components/exercises/VideoQuestion.test.js]: Is that space at the end intended?

File: src/components/exercises/VideoQuestion.test.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```
Comment (inyono) [inline src/components/exercises/VideoQuestion.test.js]: Thanks for adding a test 👍

File: src/components/exercises/VideoQuestion.js [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+    this.player.seek(0);
+    this.setState({
+      isPaused: false,
+      isFinished: false
+    });
+  };
+
+  selectAnswer = key => () => {
+    this.setState(({highlighted}) => {
+      highlighted = highlighted === key ? null : key;
```
Comment (inyono) [inline src/components/exercises/VideoQuestion.js]: You overwrite the parameter here. That's bad practice in general. You could instead do:

```.js
this.setState({ highlighted }) => ({
  highlighted: highlighted === key ? null : key
})
```

**Pull request merged**